### PR TITLE
Update to use ILP Client instead of Wallet Client

### PR DESCRIPTION
**Pull request by sharafian**

### CODE REVIEW

File: overview.html [hunk 1]
```diff
@@ -27,7 +27,7 @@ <h1 class="page-header">Docs Overview</h1>
...
+          <p>To dive into the technical specs, see the <a target="_blank" href="https://github.com/interledger/rfcs">Interledger RFCs</a>. Also see the documentation for the components of the reference implementation, code-named <a target="_blank" href="https://github.com/interledger/five-bells-demo#why-five-bells">"Five Bells"</a>. There are the <a href="{{site.interledgerjs_baseurl}}js-ilp-connector/apidoc/">connector</a>, the example <a href="{{site.interledgerjs_baseurl}}five-bells-ledger/apidoc/">ledger</a> and the application layer <a href="{{site.interledgerjs_baseurl}}five-bells-wallet/apidoc/">ILP Kit</a>.</p>
```
Comment (emschwartz) [inline overview.html]: This should be changed from wallet to `ilp-kit`

File: libraries.html [hunk 2]
```diff
@@ -15,17 +15,17 @@ <h1 class="page-header">Libraries &amp; Tools</h1>
...
 
 
-          <h2><b>(COMING SOON)</b> Low Level: ILP Client</h2>
+          <h2>Low Level: ILP Client (PSK, ILQP)</h2>
 
-          <p>Build Interledger payments into other protocols or low-level applications with the ILP Client. This library gives you direct control over the Interledger parameters and allows you to determine how to handle account discovery, condition negotiation, and communication between senders and recipients.</p>
+          <p>Build Interledger payments into other protocols or low-level applications with the ILP Client's direct <a target="_blank" href="https://github.com/interledgerjs/ilp#pre-shared-key-psk-transport-protocol">PSK</a> and <a target="_blank" href="https://github.com/interledgerjs/ilp#ilqpquoteplugin-query--promisequote">ILQP</a> functions. This library gives you direct control over the Interledger parameters and allows you to determine how to handle account discovery, condition negotiation, and communication between senders and recipients.</p>
```
Comment (emschwartz) [inline libraries.html]: What is a "direct" function?

File: overview.html (hunk 1 above)
Comment (sharafian) [inline overview.html]: Do you know how to change the ILP Kit docs to compile to an `ilp-kit` directory instead of `five-bells-wallet`?

Comment (emschwartz) [inline overview.html]: Nope

Comment (emschwartz) [inline overview.html]: Actually I take that back, it's just `{{site.interledgerjs_baseurl}}ilp-kit/apidoc` (https://interledgerjs.github.io/ilp-kit/apidoc/)

**Pull request merged**